ai/executors/pathfinder_module.py

```diff
@@ -65,47 +65,19 @@ def player_optional_obstacles(self, ball_collision: bool) -> List[Obstacle]:
         return path_obstacles
 
     def generate_path(self, player, ai_cmd: AICommand):
+
         way_points = ai_cmd.way_points
         start = player.position
         target = ai_cmd.target.position
-        path_to_target = self.sub_paths[player][-1]
         sub_paths = []
+        path_positions = [WayPoint(start), *way_points, WayPoint(target)]
+        for i in range(len(path_positions)-1):
+            sub_paths.append(self.generate_simple_path(path_positions[i].position, path_positions[i+1],
+                                                       player.velocity.position))
+        path = sub_paths[0]
+        for i in range(len(sub_paths)-1):
```

Comment:
À tester
```python
path = sum(sub_paths)
```

Reply:
jai testé ça en premier, somewhat ya une message derreur "int + Path not define"